### ISSUE TITLE
TimeAgo returns Monday even when it's not Monday

### ISSUE THREAD

**Issue by CrafterSvK** [diamondburned/gotkit#1]
I've used gtkcord4 for a while and I noticed that yesterday messages have date with `Monday at xx:xx:xx`. I found that TimeAgo returns formatted string which returns this string. 

I replaced this:
`{d: Week, s: "Monday at %X"},`
with this, as I don't really know why would it be Monday.
`{d: 2 * Day, s: "Yesterday at %X"},`

I don't really know how do you want to handle this. So I leave it up to you.
I hope you don't mind.

**Comment by diamondburned** [comment]
```diff
diff --git a/app/locale/locale.go b/app/locale/locale.go
index da45b2e..7d8d14f 100644
--- a/app/locale/locale.go
+++ b/app/locale/locale.go
@@ -129,8 +129,9 @@ type truncator struct {
 }
 
 var longTruncators = []truncator{
-       {d: Day, s: "Today at %X"},
-       {d: Week, s: "Monday at %X"},
+       {d: 1 * Day, s: "Today at %X"},
+       {d: 2 * Day, s: "Yesterday at %X"},
+       {d: Week, s: "%A at %X"},
        {d: -1, s: "%X %x"},
 }
 ```

This should work. I'll push a fix.

**Comment by diamondburned** [comment]
Fixed in 64c5f30.
